src/apps/create-patron-user-info/UserInfo.tsx

```diff
@@ -101,10 +101,18 @@ const UserInfo: FC<UserInfoProps> = ({ cpr }) => {
             />
           </div>
           <div className="patron-buttons">
-            <button type="submit" className="btn-primary btn-filled btn-small">
+            <button
+              type="submit"
+              className="btn-primary btn-filled btn-small"
+              data-cy="complete-user-registration"
```

Comment:
Maybe I would make the names a bit more specific - `complete-user-registration-button` & `cancel-user-registration-button` but otherwise I see no issues :))

Reply:
Done! Did the same changes in the sister PR in dpl-cms 👍 

I'll merge now 😄